modules/install-consul/install-consul

```diff
@@ -212,10 +216,31 @@ function install_binaries {
   sudo chmod a+x "$run_consul_dest_path"
 }
 
+function install_tls_certificates {
+  local readonly path="$1"
+  local readonly user="$2"
+  local readonly ca_file_path="$3"
+  local readonly cert_file_path="$4"
+  local readonly key_file_path="$5"
+
+  local readonly consul_tls_certs_path="$path/tls/"
+  local readonly ca_certs_path="$consul_tls_certs_path/ca/"
+
+  sudo cp "$ca_file_path" "$ca_certs_path"
+  sudo cp "$cert_file_path" "$consul_tls_certs_path"
+  sudo cp "$key_file_path" "$consul_tls_certs_path"
+
+  sudo chown -R "$user:$user" "$consul_tls_certs_path"
+  sudo chmod -R u=rX,g=,o= "$consul_tls_certs_path"
```

Comment:
To set execute permissions on the `tls/ca` directory, otherwise `-R` would only set `u=r` and botch the directory. I could do pretty much the same with `find`/`-exec` if you think that's cleaner:
```
sudo find "$consul_tls_certs_path/" -type f -exec chmod u=r,g=,o= {} \;
```

Reply:
Oh, I see, you're modifying the dir. Yea, it's probably better to modify the files within the dir to all be solely read-only for the owner. Using `find` seems like a reasonable approach. Alternatively, just call `chmod` three times, once on each file path, just as the three calls to `cp`.